### DISCUSSION TITLE
Job is about to start running on the runner

### DISCUSSION THREAD

**Discussion by tdubb123** [community/community#48281] [Actions]
### Select Topic Area

Product Feedback

### Body

i installed github action runner on my vagrant VM running ubuntu 20.04. I got it to show up in github runners. 

Then i am using the following and try to run simple job

.github/workflows/test.yaml

name: test
on:
  push
jobs:
  build:
    runs-on: self-hosted
    steps:
      - uses: actions/checkout@v3
      - name: Run a one-line script
        run: echo Hello, world!

but the job is stuck here
Requested labels: self-hosted
Job defined at: tdubb123/gitops/.github/workflows/test.yaml@refs/heads/main
Waiting for a runner to pick up this job...
Job is about to start running on the runner: k8gui (repository)

any idea?

**Comment by RobQuistNL** [comment]
Pay for their runners I suppose :shrug:

**Comment by berkon** [comment]
Did you find a solution for this issue? I have the same problem. I says that the job is about to start and the runner shows "Active" on the UI, but it never really starts.
